agents/hybrid.py

```diff
@@ -72,8 +71,9 @@ def update_q_td(self, state, action, rpe):
         :param rpe: Reward prediction error
         :return:
         """
-        alpha = self.alpha_1 if state == 0 else self.alpha_2
-        self.q_td[state, action] += alpha * rpe
+
+        # alpha = self.alpha_1 if state == 0 else self.alpha_2
```

Comment:
this part was already removed. why are you putting it back and commenting it out?

Reply:
It was't on my version!